share/locale/en.json

```diff
@@ -1,165 +1,165 @@
 {
-    "config_onion_service": "Configuring onion service on port {0:d}.",
-    "preparing_files": "Preparing files to share.",
+    "config_onion_service": "Setting up onion service on port {0:d}.",
+    "preparing_files": "Preparing OnionShare.",
     "wait_for_hs": "Waiting for HS to be ready:",
     "wait_for_hs_trying": "Trying…",
     "wait_for_hs_nope": "Not ready yet.",
-    "wait_for_hs_yup": "Ready!",
-    "give_this_url": "Give this address to the person you're sending the file to:",
-    "give_this_url_stealth": "Give this address and HidServAuth line to the person you're sending the file to:",
```

Comment:
Most everyone are unaware of the direct and speedy HidServAuth line that goes directly from non-confused user to very-confused user.

Reply:
Unfortunately I don't see a way around mentioning the HidServAuth stuff. It's an advanced (and not very user-friendly) feature, but that some advanced users choose to use anyway. You can read more about it here: https://github.com/micahflee/onionshare/wiki/Stealth-Onion-Services

In version 2.0 most people probably won't use it because it's only supported for people who choose to use legacy onion services.